.github/workflows/deploy_cms.yaml

```diff
@@ -0,0 +1,70 @@
+name: Deploy CMS
+
+on:
+  push:
+        branches: ['main']
+        # Run only if there are at least one change matching the following paths
+        paths:
+          - 'apps/strapi-cms/**'
+          - '.github/workflows/deploy_cms.yaml'
+
+  workflow_dispatch:
+    inputs:
+      logLevel:
+        description: 'Log level'
+        required: true
+        default: 'warning'
+        type: choice
+        options:
+        - warning
+      environment:
+        description: 'The environment used as target'
+        type: choice
+        required: true
+        default: prod
+        options:
+        - prod
+
+defaults:
+  run:
+    shell: bash
+    working-directory: apps/strapi-cms
+
+permissions:
+  id-token: write   # This is required for requesting the JWT
+  contents: read    # This is required for actions/checkout
+
+jobs:
+  deploy:
+    name: Build and push CMS image
+    runs-on: ubuntu-latest
+    continue-on-error: false
+    environment: ${{ inputs.environment }}
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@3df4ab11eba7bda6032a0b82a6bb43b11571feac # v4.0.0
+    
+
+      - name: Configure AWS Credentials
+        uses: ./.github/actions/configure-aws-credentials
+        with:
+          aws_region: ${{ env.AWS_REGION || 'eu-south-1' }}
+          role_to_assume: ${{ secrets.IAM_ROLE }}
```

Comment:
we could create a new IAM_ROLE to follow [the least privilege principle](https://en.wikipedia.org/wiki/Principle_of_least_privilege)

Reply:
ok, i have open a PR https://github.com/pagopa/b2b-portals/pull/58 for create new role: 
1) GitHubActionDeployECS for deploy CMS on ECS. In Workflow i have write secrets.IAM_ROLE_DEPLOY_CMS
2) GitHubActionDeployWebsite for deploy a static website on S3. In Workflow i have write secrets.IAM_ROLE_DEPLOY_WEBSITE